android/app/src/main/java/it/teamdigitale/app/italiaapp/MainActivity.java

```diff
@@ -71,6 +76,14 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
       super.onRequestPermissionsResult(requestCode, permissions, grantResults);
     }
 
+    // Setup badge library showoff
+    protected List<ReactPackage> getPackages() {
```

Comment:
Usually libraries get added to MainApplication. Are you sure of this change?

Reply:
I followed literally instructions here, but yes, after figuring out this project setup I moved code in the right place in 3ca589af